frontend/public/components/resource-quota.jsx

```diff
@@ -35,64 +55,125 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
-    <div className="col-sm-3 col-xs-3">{used}</div>
-    <div className="col-sm-3 col-xs-3">{max}</div>
+    <div className="col-sm-2 hidden-xs co-resource-quota-icon"><ProportionIcon percent={proportions.percentUsed} /></div>
+    <div className="col-sm-3 col-xs-3">{proportions.used}</div>
+    <div className="col-sm-3 col-xs-3">{proportions.max}</div>
   </div>;
 };
 
-const Details = ({obj: rq}) => <React.Fragment>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Resource Quota Overview" />
-    <ResourceSummary resource={rq} podSelector="spec.podSelector" showNodeSelector={false} />
-  </div>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Resource Usage" />
-    <div className="co-m-table-grid co-m-table-grid--bordered">
-      <div className="row co-m-table-grid__head">
-        <div className="col-sm-4 col-xs-6">Resource Type</div>
-        <div className="col-sm-2 hidden-xs">Capacity</div>
-        <div className="col-sm-3 col-xs-3">Used</div>
-        <div className="col-sm-3 col-xs-3">Max</div>
+const QuotaScopes = ({quota}) => {
+  const scopes = _.get(quota, ['spec', 'scopes']);
+  if (!scopes) {
+    return <dd>-</dd>;
+  }
+  const retVal = scopes.map(scope => {
+    const scopeObj = _.get(quotaScopes, scope);
+    return scopeObj ?
+      <dd key={scope}>
+        <div className="co-resource-quota-scope__label">{scopeObj.label}</div>
+        <div className="co-resource-quota-scope__description">{scopeObj.description}</div>
+      </dd>
+      : <dd key={scope} className="co-resource-quota-scope__label">{scope}</dd>;
+  });
+  return retVal;
+};
+
+const showCharts = resourceTypes => {
+  const chartResourceTypes = ['requests.cpu', 'limits.cpu', 'requests.memory', 'limits.memory'];
+  return _.intersection(resourceTypes, chartResourceTypes).length > 0;
+};
+
+const Details = ({obj: rq}) => {
+  const resourceTypes = getResourceTypes(rq);
+  const showChartRow = showCharts(resourceTypes);
+  const popover = (
+    <Popover>
+      Requests are the amount of resources you expect to use. These are used when establishing if the cluster can fulfill your Request.<br />
+      <br />
+      Limits are a maximum amount of a resource you can consume. Applications consuming more than the Limit may be terminated.<br />
+      <br />
+      A cluster administrator can establish limits on both the amount you can Request and your Limits with a &nbsp;
+      <a href={helpLink('dev_guide/compute_resources.html#dev-compute-resources')} target="_blank" rel="noopener">Resource Quota</a>.
+    </Popover>
+  );
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Resource Quota Overview" />
+      {showChartRow && <div className="co-resource-quota-chart-row">
+        {_.includes(resourceTypes, 'requests.cpu') &&
+          <div className="co-resource-quota-chart">
+            <Gauge title="CPU Request" percentUsed={getProportions(rq, 'requests.cpu').percentUsed} />
```

Comment:
We should probably warn at 90% and only show error if >100%

Have you tested quota usage >100%?

Reply:
Ok, set thresholds to `{warn: 90,error: 100};`. >100% looks like:
![image](https://user-images.githubusercontent.com/12733153/49452699-aa1a7c00-f7af-11e8-9982-d1001dbbb910.png)
Should `error` be somewhere between 95 - 99 since when at 100 the thin error threshold arc doesn't display -only the thin ok and warning arcs show when error = 100
![image](https://user-images.githubusercontent.com/12733153/49453197-a76c5680-f7b0-11e8-9e90-92db07a07c91.png)
![image](https://user-images.githubusercontent.com/12733153/49453168-97547700-f7b0-11e8-846a-a437e975255c.png)

cc @cshinn

Reply:
Decided in scrum to keep `{warn: 90,error: 100}`, so no error threshold arc